### DISCUSSION TITLE
Review and ATS Score for my resume - Vaishali Sharma

### DISCUSSION THREAD

**Discussion by vaishali-sharma-20** [recodehive/Stackoverflow-Analysis#356] [Q&A]
Hello @sanjay-kv sir ,
My name is Vaishali Sharma, and I am writing to request your assistance in reviewing my resume and providing an ATS  score. I am currently preparing for my job search and want to ensure that my resume is optimized both in content and format to meet industry standards.

Please review my resume for two different Profile 
I colored my resume for the Business Analyst Profile - [Vaishali Sharma Resume .pdf](https://github.com/user-attachments/files/16356872/Vaishali.Sharma.Resume.pdf)
for Software Engineering -  [Resume - Vaishali Sharma.pdf](https://github.com/user-attachments/files/16356888/Resume.-.Vaishali.Sharma.pdf)

Your feedback and insights will be invaluable in helping me improve my resume to better align with job requirements and increase my chances of securing interviews.

Thank you in advance for your time and assistance. I look forward to your response.

**Comment by sanjay-kv** [comment]
[ATS Version of CV (16).docx](https://github.com/user-attachments/files/16357178/ATS.Version.of.CV.16.docx)
[BA Vais.pdf](https://github.com/user-attachments/files/16357179/BA.Vais.pdf)

**Comment by sanjay-kv** [comment]
[ATS Version of CV (17).docx](https://github.com/user-attachments/files/16357233/ATS.Version.of.CV.17.docx)
[sde vs.pdf](https://github.com/user-attachments/files/16357234/sde.vs.pdf)
SDE role reviewed

**Comment by garv999** [comment]
Hi Vaishali,

Thank you for reaching out and sharing your resumes for review. I appreciate your initiative in preparing thoroughly for your job search.

I'll go through both versions — one tailored for the Business Analyst profile and the other for the Software Engineering role. I’ll review them for:

ATS compatibility (formatting, keywords, structure)

Content relevance to the respective roles

Clarity and impact of language

Suggestions for improvement in both design and wording

Once I’ve completed the review, I’ll share feedback along with an estimated ATS score and recommendations to enhance alignment with industry expectations.

Looking forward to helping you strengthen your applications!
Best regards,
Garv Agarwal
